### DISCUSSION TITLE
Local installation partially working

### DISCUSSION THREAD

**Discussion by PeroPonneso** [simeydotme/pokemon-cards-css#46] [Q&A]
Hi @simeydotme ,

beautiful effects you got there.

Wanted to understand better what you did in the code so I tried to run it locally.
Installed the node version stated in the requirements and run the npm install; it works but i can see only a couple effects applying on the cards.
Do I need to run something special? Maybe the foils fetch API is not working?

![image](https://github.com/simeydotme/pokemon-cards-css/assets/77463786/8b03399f-e986-4894-a008-0f5c68d9e3fd)

Or maybe I'm missing the upscaled foils for some other reason?

![image](https://github.com/simeydotme/pokemon-cards-css/assets/77463786/bb307bc3-fc6e-4b5d-98f8-ebe42f722cde)

Any tip is much appreciated

**Comment by simeydotme** [comment]
Hi, yes, the images for the foils are hosted on my CDN, they are too many and large to put in GitHub and I don't want to provide them for scraping.

If you want to test some of the foil effects you can inspect the demo page and grab the foils from the resource panel :)

if you're interested in doing something more advanced and permanent with foil images, and pokemon cards specifically, you could consider to join the malie.io discord group and ask about the foils in there (thats where I got them from originally)
